guides/source/upgrading_ruby_on_rails.md

```diff
@@ -1326,7 +1326,7 @@ user.highlights.first.filename # => "funky.jpg"
 user.highlights.second.filename # => "town.jpg"
 ```
 
-Existing applications can opt in to this new behavior by setting [`config.active_storage.replace_on_assign_to_many`][] to `true`. The old behavior will be deprecated in Rails 7.0 and removed in Rails 7.1.
+Existing applications can opt in to this new behavior by setting [`config.active_storage.replace_on_assign_to_many`][] to `true`. The old behavior was deprecated in Rails 7.1 and removed in Rails 7.2.
```

Comment:
The existing text is correct here. `replace_on_assign_to_many` the _config value_ was deprecated in 7.1 and removed in 7.2, but the _behavior_ was deprecated in 7.0 and removed in 7.1 (meaning the config value had no effect in 7.1) as documented.

Reply:
My bad, I see you are correct. I'll cancel this portion. Thanks 👍